src/index.ts

```diff
@@ -225,11 +225,17 @@ const handleClearKey = async (ctx: Context, _request?: Request) => {
 		}
 	}
 	const toDeleteArray = [...toDelete];
-	await ctx.bucket.ISSUANCE_KEYS.delete(toDeleteArray);
+
+	if (toDeleteArray.length > 0) {
+		await ctx.bucket.ISSUANCE_KEYS.delete(toDeleteArray);
+	}
 
 	ctx.waitUntil(clearDirectoryCache());
 
-	return new Response(`Keys cleared: ${toDeleteArray.join('\n')}`, { status: 201 });
+	return new Response(
+		`Keys cleared: ${toDeleteArray.length > 0 ? toDeleteArray.join('\n') : 'None'}`,
```

Comment:
nits: if there is nothing to delete, response can be returned earlier. this avoid a complex expression with ternary embedded into a string template